### PR TITLE
Add react-tag-names to "related" section

### PR DESCRIPTION
**Pull request by jgierer12**
I maintain a similar list of React's supported tag names.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -66,6 +66,8 @@ Yields:
     — List of MathML tags
 *   [`svg-tag-names`](https://github.com/wooorm/svg-tag-names)
     — List of SVG tags
+*   [`react-tag-names`](https://github.com/jgierer12/react-tag-names)
+    — List of React's HTML and SVG tag names
```
Comment (wooorm) [inline readme.md]:
```suggestion
    — List of React’s HTML and SVG tag names
```

**Review by wooorm — CHANGES_REQUESTED** [review]
Looks good, could you use a curly apostrophe though?

File: readme.md (hunk 1 above)
Comment (jgierer12) [reply readme.md]: Sure :+1:

**Review by wooorm — APPROVED** [review]

**Comment by wooorm** [comment]
Thanks @jgierer12!

**Pull request merged**